### ISSUE TITLE
Suggestions

### ISSUE THREAD

**Issue by Arnemd** [Rajat-dhyani/Stock-Price-Predictor#1]
Nice project, however, I have some suggestions. Your test set is completely included in the training set. So you can't really use it to evaluate the predictive power of your model. You kind of tackled this by your robustness check, which is actually the validation check on an independent test set. 

Also, you mixed the two arguments in the plot functions. The plot functions take as arguments (actual, prediction), but in the notebook you pluged in (prediction, actual). So actually the green line, that is lagging behind the more volatile line, is the prediction.
